src/app.ts

```diff
@@ -133,6 +142,16 @@ async function app() {
 
   const tank = new FSHTank(docs, config);
   await Promise.all(dependencyDefs);
+
+  // Check for StructureDefinition
+  const structDef = defs.fishForFHIR('StructureDefinition', Type.Resource);
+  if (!(structDef && structDef.version === '4.0.1')) {
+    logger.error(
+      'StructureDefinition resource not found for 4.0.1. Your FHIR package may be corrupt.'
```

Comment:
It may be helpful to point the user to the `.fhir` cache here.

Reply:
Good idea!